packages/fluentui/react-northstar/src/components/Form/FormField.tsx

```diff
@@ -65,120 +72,166 @@ export type FormFieldStylesProps = Required<Pick<FormFieldProps, 'type' | 'inlin
   hasErrorMessage: boolean;
 };
 
-const FormField: React.FC<WithAsProp<FormFieldProps>> & FluentComponentStaticProps<FormFieldProps> = props => {
-  const context: ProviderContextPrepared = React.useContext(ThemeContext);
-  const { setStart, setEnd } = useTelemetry(FormField.displayName, context.telemetry);
-  setStart();
-
-  const {
-    children,
-    control,
-    id,
-    label,
-    message,
-    name,
-    required,
-    type,
-    className,
-    design,
-    styles,
-    variables,
-    inline,
-    errorMessage,
-  } = props;
-
-  const ElementType = getElementType(props);
-  const unhandledProps = useUnhandledProps(FormField.handledProps, props);
-  const messageId = React.useRef<string>();
-  messageId.current = getOrGenerateIdFromShorthand('error-message-', message || errorMessage, messageId.current);
-  const labelId = React.useRef<string>();
-  labelId.current = getOrGenerateIdFromShorthand('form-label-', id, labelId.current);
-
-  const getA11yProps = useAccessibility<FormFieldBehaviorProps>(props.accessibility, {
-    debugName: FormField.displayName,
-    mapPropsToBehavior: () => ({
-      hasErrorMessage: !!errorMessage,
-      messageId: messageId.current,
-      labelId: labelId.current,
-    }),
-    rtl: context.rtl,
-  });
-
-  const { classes, styles: resolvedStyles } = useStyles<FormFieldStylesProps>(FormField.displayName, {
-    className: formFieldClassName,
-    mapPropsToStyles: () => ({
-      type,
-      inline,
+const FormField = compose<'div', FormFieldProps, FormFieldStylesProps, {}, {}>(
```

Comment:
As we agreed to use `FormCustomField` let's revert changes in this file as we don't need to introduce `compose()` there